### PR TITLE
added code of conduct

### LINKED ISSUE #381: Fix Grammatical issues in Code of Conduct and add it in docs
🐞 **Problem**

The language in the COC is flawed.

```
 We will take action when members of our community violate this code such as contacting einsteinpy.project@gmail.com (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.
```

🎯 **Goal**

Fix it,

And add the COC to the documentation as well.

💡 **Possible solutions**

📋  **Steps to solve the problem**

 * Comment below about what you've started working on.
 * Add, commit, push your changes
 * Submit a pull request and add this in comments - `Addresses #<put issue number here>`
 * Ask for a review in comments section of pull request
 * Celebrate your contribution to this project 🎉

### PR DESCRIPTION
**Pull request by ShamanthNyk**
Fixes #381

### CODE REVIEW

**Comment by shreyasbapat** [comment]
The change should be this: 
https://matrix.to/#/!KXuIZXQfZbpAMwZnrm:matrix.org/$1579888796203773ebMkv:matrix.org?via=matrix.org&via=converser.eu&via=cybre.space

File: CODE_OF_CONDUCT.rst [hunk 1]
```diff
@@ -42,7 +42,7 @@ As members of the community,
 * We pledge to help the entire community follow the code of conduct, and to
   not remain silent when we see violations of the code of conduct. We will
   take action when members of our community violate this code such as
-  contacting einsteinpy.project@gmail.com (all emails sent to this
+  contacting shreyas@einsteinpy.org (all emails sent to this
   address will be treated with the strictest confidence) or talking privately
   with the person.
```
Comment (shreyasbapat) [inline CODE_OF_CONDUCT.rst]: We pledge to help the entire community follow the code of conduct, and to not remain silent when we see violations of the code of conduct. We will take action when members of our community violate this code such as contacting shreyas@einseinpy.org (all emails sent to this address will be treated with the strictest confidence) or talking privately with the person.

Comment (shreyasbapat) [reply CODE_OF_CONDUCT.rst]: Kindly replace these lines with the ones I suggested.

**Comment by shreyasbapat** [comment]
Also, you will need to squash the commits into one! 
And probaly rebase your branch as well.

**Comment by shreyasbapat** [comment]
Correction -> Rebase not required!

File: docs/source/codeofconduct.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
+﻿Code of Conduct for the einsteinpy community
```
Comment (shreyasbapat) [inline docs/source/codeofconduct.rst]:
```suggestion
﻿Code of Conduct for the einsteinpy community
```
Code of Conduct

**Comment by shreyasbapat** [comment]
@ShamanthNyk Are you still on it?

**Comment by ShamanthNyk** [comment]
Yes

**Comment by rohitsanj** [comment]
Fixing things up with @ShamanthNyk

**Pull request closed without merging**